brian2tools/plotting/base.py

```diff
@@ -85,6 +87,18 @@ def brian_plot(brian_obj,
         if 'var_unit' not in kwds and isinstance(values, Quantity):
             kwds['var_unit'] = values._get_best_unit()
         return plot_state(brian_obj.t, values, axes=axes, **kwds)
+    elif isinstance(brian_obj, StateMonitorView):
+        monitor = brian_obj.monitor
+        if len(monitor.record_variables) != 1:
+            raise TypeError('brian_plot only works for a StateMonitor that '
+                            'records a single variable.')
```

Comment:
Doesn't have to be in this release but maybe a nice feature for the future? Could do shared x-axis so if you move one graph all the others move too. I often use this when debugging models.